plugins/source/aws/resources/services/kms/key_policies.go

```diff
@@ -0,0 +1,82 @@
+package kms
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/kms"
+	"github.com/aws/aws-sdk-go-v2/service/kms/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+type KeyPolicy struct {
+	Name   string
+	Policy *string
+}
+
+func KeyPolicies() *schema.Table {
+	return &schema.Table{
+		Name:        "aws_kms_key_policies",
+		Description: `https://docs.aws.amazon.com/kms/latest/APIReference/API_GetKeyPolicy.html`,
+		Resolver:    fetchKeyPolicies,
+		Transform:   transformers.TransformWithStruct(&KeyPolicy{}),
+		Multiplex:   client.ServiceAccountRegionMultiplexer("kms"),
+		Columns: []schema.Column{
+			{
+				Name:     "account_id",
+				Type:     schema.TypeString,
+				Resolver: client.ResolveAWSAccount,
+			},
+			{
+				Name:     "region",
+				Type:     schema.TypeString,
+				Resolver: client.ResolveAWSRegion,
+			},
+			{
+				Name:     "key_arn",
+				Type:     schema.TypeString,
+				Resolver: schema.ParentColumnResolver("arn"),
+				CreationOptions: schema.ColumnCreationOptions{
+					PrimaryKey: true,
+				},
+			},
+			{
+				Name:     "name",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("Name"),
+				CreationOptions: schema.ColumnCreationOptions{
+					PrimaryKey: true,
+				},
+			},
+			{
+				Name:     "policy",
+				Type:     schema.TypeJSON,
+				Resolver: schema.PathResolver("Policy"),
+			},
+		},
+	}
+}
+
+func fetchKeyPolicies(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
+	c := meta.(*client.Client)
+	svc := c.Services().Kms
+
+	const policyName = "default"
```

Comment:
Quite weird that the only valid name is `default` 🤷

Reply:
That's what the doc says... We could've left it off to add it later if needed maybe (but then the PK change would've been breaking)